content/deploy/maintain/audit.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Archive the PostgreSQL Audit trail'
+date: '2024-03-26T11:20:56-03:00'
+categories: ["how-to"]
+description: How to archive a partition of the Audit trail on your Rhize deployment
+weight: 300
+menu:
+  main:
+    parent: maintain
+    identifier:
+---
+
+The [audit trial]({{< relref "/how-to/audit" >}}) can generate a high volume of data, so it is a good practice to periodically _archive_ portions of it. An archive separates a portion of the data from the database and keeps it for long-term storage. This process involves the use of PostgresQL [Table Partitions](https://www.postgresql.org/docs/current/ddl-partitioning.html).
+
+## Prerequisites
+
+Before you start, ensure you have the following:
+
+- A designated backup location, for example `~/rhize-backups/libre-audit`.
+- Access to the [Rhize Kubernetes Environment](/deploy/install/setup-kubernetes)
+{{% param pre_reqs %}}
+
+
+Also, before you start, confirm you are in the right context and namespace.
+
+{{% param k8s_cluster_ns %}}
+
+## Steps
+
+To archive the PostgreSQL Audit trail, follow these steps:
+
```

Comment:
Before detatching / dumping, provide instruction on how to list the current partitions:

1. Based on your retention period query for the names of the existing partitions:

   ```bash
   kubectl exec -i audit-postgresql-0 -- psql -h localhost \ 
     -d audit -U <DB_USER> \
     -c 'SELECT * FROM x'
   ```
   
    Record the name `<PARTITION_NAME>` of the partition you wish to detach and archive.